python/lsst/meas/algorithms/loadReferenceObjects.py

```diff
@@ -290,146 +292,157 @@ def _makeBoxRegion(BBox, wcs, BBoxPadding):
+        reference catalogs which intersect but are not fully contained within
+        the input region are further filtered by the specified filter function.
+        This function returns a single source catalog containing all reference
+        objects inside the specified region.
 
         Parameters
         ----------
         region : `lsst.sphgeom.Region`
-            This can be any type that is derived from `lsst.sphgeom.region` and should
-            define the spatial region for which reference objects are to be loaded.
-        filtFunc : callable
-            This optional parameter should be a callable object that takes a reference
-            catalog and its corresponding region as parameters, filters the catalog by
-            some criteria and returns the filtered reference catalog. If the value is
-            left as the default (None) than an internal filter function is used which
-            filters according to if a reference object falls within the input region.
-        filterName : `str`
-            Name of camera filter, or None or blank for the default filter
-        epoch : `astropy.time.Time` (optional)
-            Epoch to which to correct proper motion and parallax,
-            or None to not apply such corrections.
+            This can be any type that is derived from `lsst.sphgeom.Region` and
+            should define the spatial region for which reference objects are to
+            be loaded.
+        filtFunc : callable or `None`, optional
+            This optional parameter should be a callable object that takes a
+            reference catalog and its corresponding region as parameters,
+            filters the catalog by some criteria and returns the filtered
+            reference catalog. If `None`, an internal filter function is used
+            which filters according to if a reference object falls within the
+            input region.
+        filterName : `str` or `None`, optional
+            Name of camera filter, or `None` or blank for the default filter.
+        epoch : `astropy.time.Time` or `None`, optional
+            Epoch to which to correct proper motion and parallax, or `None` to
+            not apply such corrections.
 
         Returns
         -------
         referenceCatalog : `lsst.afw.table.SourceCatalog`
             Catalog containing reference objects which intersect the input region,
-            filtered by the specified filter function
+            filtered by the specified filter function.
 
         Raises
         ------
-        `lsst.pex.exception.RuntimeError`
-            Raised if no reference catalogs could be found for the specified region
-
-        `lsst.pex.exception.TypeError`
-            Raised if the loaded reference catalogs do not have matching schemas
-
+        RuntimeError
+            Raised if no reference catalogs could be found for the specified
+            region.
+        TypeError
```

Comment:
This is actually a `pex.exceptions.TypeError` If you dont want it to be, we need to change the code.

Reply:
Like I said in the other comment I am happy for them to be changed as you did. The reason that this was a `pexExceptions.TypeError` was because we were raising it explicitly here: https://github.com/lsst/meas_algorithms/blob/master/python/lsst/meas/algorithms/loadReferenceObjects.py#L460 before the change (that is unless you changed it before the force push too, and I somehow was mistakenly looking at master, which is something I would do) and I was thinking it should be that type in `loadPixelBox` as well because that calls this method and it could have been raised here. That said its all moot because you changed the raise.